gems/manageiq_ansible_tower/manageiq_ansible_tower.gemspec

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'manageiq_ansible_tower/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "manageiq_ansible_tower"
+  spec.version       = ManageiqAnsibleTower::VERSION
+  spec.authors       = ["Brandon Dunne"]
+  spec.email         = ["bdunne@redhat.com"]
+
+  spec.summary       = %q{Ansible Tower Provider for ManageIQ}
+  spec.description   = %q{Ansible Tower Provider for ManageIQ}
```

Comment:
I'm only (somewhat) dubious because we currently seem to have a couple of "the generic library to do the thing" gems, which only end up used by our app, and which we feel free to change as needed by our app. To me a suitably simple REST API should *be* the abstraction layer, without necessarily requiring an in-app object layer on top of our own. But, I'm not looking at the actual thing, so really just raising the question rather than expressing a strong opinion.